packages/synapse-react-client/src/components/ChallengeSubmission/SubmissionDirectoryList.tsx

```diff
@@ -458,10 +458,7 @@ function SubmissionDirectoryList({
             To learn more about how to create and submit the Docker containers
             using command line, see our{' '}
             <Link
-              to={{
```

Comment:
Since this is an external link, we don’t need to use an object at all - just pass the full URL as a string.